lib/App/MintTag.pm

```diff
@@ -141,7 +141,10 @@ sub ensure_initial_prep ($self) {
     die "local path $dir does not exist! (maybe you should set clone = true)\n"
       unless $self->config->should_clone;
 
-    chdir $dir->parent or die "Couldn't chdir to $dir\'s parent!\n";
+    unless (chdir $dir->parent) {
+      Path::Tiny::path($dir->parent)->mkpath;
```

Comment:
`->parent` on a Path::Tiny object gives you another Path::Tiny, so the `path()` here is redundant; you can just write `$dir->parent->mkpath`.